src/main/java/com/minecolonies/colony/buildings/BuildingMiner.java

```diff
@@ -347,13 +348,33 @@ public void readFromNBT(NBTTagCompound compound)
         startingLevelNode = compound.getInteger(TAG_SN);
 
         NBTTagList levelTagList = compound.getTagList(TAG_LEVELS, Constants.NBT.TAG_COMPOUND);
-        for(int i = 0; i < levelTagList.tagCount(); i++)
+        for (int i = 0; i < levelTagList.tagCount(); i++)
         {
             Level level = Level.createFromNBT(levelTagList.getCompoundTagAt(i));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'level' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)